pkg/devfile/adapters/common/types.go

```diff
@@ -34,6 +34,13 @@ type BuildParameters struct {
 	IgnoredFiles    []string                // IgnoredFiles is the list of files to not push up to a component
 }
 
+// DeployParameters is a struct containing the parameters to be used when building the image for a devfile component
+type DeployParameters struct {
+	Path            string                  // Path refers to the parent folder containing the source code to push up to a component
```

Comment:
do we need the path field?

Reply:
Possibly not once we have the code to correctly fetch the manifest. 
It just gets used at present to know where to read the file from on disk.